src/OrchardCore/OrchardCore.Infrastructure/Scripting/CommonGeneratorMethods.cs

```diff
@@ -50,7 +50,12 @@ static CommonGeneratorMethods()
             using var uncompressedStream = MemoryStreamFactory.GetStream((int)compressedStream.Length);
             gZip.CopyTo(uncompressedStream);
 
-            return Convert.ToBase64String(uncompressedStream.GetBuffer(), 0, (int)uncompressedStream.Length);
+            var readOnlySequence = uncompressedStream.GetReadOnlySequence();
+            var base64String = readOnlySequence.IsSingleSegment ?
```

Comment:
I don't think that is necessary because the stream will do it already: https://github.com/microsoft/Microsoft.IO.RecyclableMemoryStream/blob/master/src/RecyclableMemoryStream.cs#L508

Reply:
Yes, you're right. It would only make sense if `ReadOnlySequence` could be used instead.